SPECS/kernel/config_aarch64

```diff
@@ -7568,6 +7568,7 @@ CONFIG_HID_THRUSTMASTER=m
 CONFIG_THRUSTMASTER_FF=y
 CONFIG_HID_UDRAW_PS3=m
 CONFIG_HID_U2FZERO=m
+# CONFIG_HID_UNIVERSAL_PIDFF is not set
```

Comment:
Result of https://github.com/microsoft/CBL-Mariner-Linux-Kernel/commit/c1fde337b317f0a226de92803288741c30799eb0